org.briarproject.Briar.yaml

```diff
@@ -0,0 +1,63 @@
+app-id: org.briarproject.Briar
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.openjdk17
+command: briar
+rename-icon: briar
+finish-args:
+  - --socket=x11
+  - --share=ipc
+  - --share=network
+  # I'll suggest to use XDG_CONFIG_HOME
+  - --persist=~/.briar
+  - --filesystem=xdg-download
+  - --env=PATH=/usr/bin:/app/bin:/app/jre/bin
+  - --env=JAVA_HOME=/app/jre
+
+build-options:
+  env:
+    JAVA_HOME: /usr/lib/sdk/openjdk17/
+modules:
+  - name: openjdk
+    buildsystem: simple
+    build-commands:
+      - /usr/lib/sdk/openjdk17/install.sh
+
+
+  - name: briar-binary
+    buildsystem: simple
+    build-commands:
+        - install -D briar.sh /app/bin/briar
+        - install -D briar.jar /app/bin/briar.jar
+        - install -D org.briarproject.Briar.desktop /app/share/applications/org.briarproject.Briar.desktop
+        - install -D org.briarproject.Briar.appdata.xml /app/share/metainfo/org.briarproject.Briar.appdata.xml
+        - install -D org.briarproject.Briar.svg /app/share/icons/hicolor/scalable/apps/briar.svg
+
+    sources:
+      - type: file
+        path: briar.sh
+
+      - type: file
+        path: org.briarproject.Briar.appdata.xml
+
+      # The .desktop file could be extracted from the released deb package
```

Comment:
then it should. We shouldn't maintain it here.

Reply:
Yeah, I contacted upstream about offline building instructions. I hope that this will yield the desktop file.
Do you want to block distribution on Flathub for as long as the desktop is not generated? Or is it acceptable to file a bug once the repository has created to not forget about it?